wat.go

```diff
@@ -4,13 +4,73 @@ import (
 	"fmt"
 	"io"
 	"log"
+	"os"
 	"os/exec"
+	"path/filepath"
 
 	"gopkg.in/fsnotify.v1"
 )
 
+type RecursiveWatcher struct {
```

Comment:
May be I should put this `RecursiveWatcher struct` in different file. Let me know if that is fine.

Reply:
No problem with it 👍